spec/models/poly_metadatum_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
```

Comment:
Some matchers I didn't know about here 😃

Reply:
I'm learning a lot asking code pilot to suggest 'better syntax' for sections of code(!)